pkgs/development/python-modules/fiona/default.nix

```diff
@@ -14,24 +15,35 @@ buildPythonPackage rec {
 
   CXXFLAGS = stdenv.lib.optionalString stdenv.cc.isClang "-std=c++11";
 
+  nativeBuildInputs = [
+    gdal # for gdal-config
```

Comment:
@FRidh Just wanted to let you know that I believe this is a situation where a package needs to be in `nativeBuildInputs` as well as `buildInputs`.

Reply:
Probably yes. `gdal-config` requires it to be in `nativeBuildInputs`, but we also build against it so also in `buildInputs`.